core/src/main/resources/FrankFrameworkConfigurationContext.xml

```diff
@@ -10,6 +11,13 @@
 		scope="singleton"
 	/>
 
+	<bean
+		name="adapterManager"
+		class="nl.nn.adapterframework.configuration.AdapterManager"
+		autowire="byType"
+		scope="singleton"
```

Comment:
Singleton betekent hier ``uniek voor een configuratie``, denk ik. Dat lijkt me wel handig om in een stukje commentaar te zetten.